GenumKit/Parsers/AssetsCatalogParser.swift

```diff
@@ -41,34 +59,41 @@ private enum AssetCatalog: String {
 extension AssetsCatalogParser {
   static let imageSetExtension = "imageset"
 
-  fileprivate func processCatalog(items: [[String: AnyObject]], withPrefix prefix: String = "") {
+  fileprivate func process(items: [[String: Any]], withPrefix prefix: String = "") -> [Entry] {
+    var result = [Entry]()
+
     for item in items {
       guard let filename = item[AssetCatalog.filename.rawValue] as? String else { continue }
```

Comment:
Yup. That's how I namespace all my constants in my projects now, and have been loving it!

Like I very often have stuff like this in my projects:

```swift
struct SomeModel {
  enum JSONKeys {
    static let firstname = "firstName"
    static let lastname = "lastName"
    static let dob = "dateOfBirth"
  }

  let firstname: String
  let lastname: String
  let dateOfBirth: String

  init?(json: [String: Any]) {
    guard
      let fn = json[JSONKeys.firstname] as? String,
      let ln = json[JSONKeys.lastname] as? String,
      let dob = json[JSONKeys.dob] as? String
      else { return nil }
    self.firstname = fn
    self.lastname = ln
    self.dateOfBirth = dob
  }
}
```